forge-gui/res/cardsfolder/h/hunting_velociraptor.txt

```diff
@@ -3,6 +3,6 @@ ManaCost:2 R
 Types:Creature Dinosaur
 PT:3/2
 K:First Strike
-S:Mode$ Continuous | Affected$ Dinosaur.YouCtrl | AddKeyword$ Prowl:2 R | AffectedZone$ Hand | Description$ Dinosaur spells you cast have prowl {2}{R}. (You may cast a spell for its prowl cost if you dealt combat damage to a player this turn with a creature with any of its creature types.)
+S:Mode$ Continuous | Affected$ Dinosaur.YouCtrl | AddKeyword$ Prowl:2 R | AffectedZone$ Stack | Description$ Dinosaur spells you cast have prowl {2}{R}. (You may cast a spell for its prowl cost if you dealt combat damage to a player this turn with a creature with any of its creature types.)
```

Comment:
Just as an aside: I discovered that this ability didn't work because I was playing my Jurassic Park commander deck and couldn't cast Pantlaza, Sun-Favored from the Command Zone. I'm chuffed that it's working as intended, now! 😊